src/pipeline.ts

```diff
@@ -0,0 +1,115 @@
+import { Client } from "./client";
+import { processSearchResponse } from "./constructors";
+import { request, RequestError } from "./lib/request";
+import { Results } from "./results";
+import { Session } from "./session";
+import { Values } from "./types";
+
+/** UserAgent to be sent along with requests identifying this sdk as the one making contact. */
```

Comment:
Probably don't need to comment this (or export it given that you can't change it?).  This is really for internal-only use. Can discuss.